src/vs/platform/mcp/common/mcpManagementService.ts

```diff
@@ -232,7 +232,7 @@ export abstract class AbstractMcpResourceManagementService extends Disposable {
 		}
 	}
 
-	protected toScannedMcpServerAndInputs(manifest: IMcpServerManifest, packageType?: PackageType): { config: IMcpServerConfiguration; inputs?: IMcpServerVariable[] } {
+	static toScannedMcpServerAndInputs(manifest: IMcpServerManifest, packageType?: PackageType): { config: IMcpServerConfiguration; inputs?: IMcpServerVariable[] } {
```

Comment:
@joelverhagen wondering why this is made public and static? This is an internal implementation of handling gallery mcp server installations. Now it is being reused for other installation flows which will make difficult to do any changes/enhancements for gallery installation flows.